lib/FieldType/Mapper/ISBNFormMapper.php

```diff
@@ -55,7 +75,42 @@ public function mapFieldDefinitionForm(FormInterface $fieldDefinitionForm, Field
     }
 
     /**
+     * @param FormInterface $fieldForm
+     * @param FieldData $data
+     */
+    public function mapFieldValueForm(FormInterface $fieldForm, FieldData $data)
+    {
+        $fieldDefinition = $data->fieldDefinition;
+        $formConfig = $fieldForm->getConfig();
+        $names = $fieldDefinition->getNames();
+        $label = $fieldDefinition->getName($formConfig->getOption('languageCode')) ?: reset($names);
+
+        $fieldForm
+            ->add(
+                $formConfig->getFormFactory()->createBuilder()
+                    ->create(
+                        'value',
+                        TextType::class,
+                        [
+                            'required' => $fieldDefinition->isRequired,
+                            'label' => $label,
```

Comment:
no `pattern` (https://github.com/ezsystems/hybrid-platform-ui/pull/65/files#diff-46b5c70bde48cbbdc0324e8043b3be5cR726) ? no way to set it with SF TextType ?